api/model/storage/postgres/dataset.go

```diff
@@ -334,6 +354,9 @@ func (s *Storage) AddVariable(dataset string, storageName string, varName string
 	if !found {
 		// add the empty column to the base table and the explain table
 		sql := fmt.Sprintf("ALTER TABLE %s_base ADD COLUMN \"%s\" TEXT;", storageName, varName)
+		if len(defaultVal) > 0 {
+			sql = fmt.Sprintf("ALTER TABLE %s_base ADD COLUMN \"%s\" TEXT DEFAULT '%s';", storageName, varName, defaultVal[0])
```

Comment:
Similar to the above comment, I would suggest against using the ... operator as some type of optional parameter passing approach. I would also suggest the if should set a variable that is used in the sql statement creation to minimize duplication.